electrum/gui/qml/java_classes/org/electrum/qr/SimpleScannerActivity.java

```diff
@@ -16,18 +25,53 @@
 import com.google.zxing.Result;
 import com.google.zxing.BarcodeFormat;
 
+import org.electrum.electrum.res.R; // package set in build.gradle
```

Comment:
Ah, nice solution in https://github.com/accumulator/python-for-android/commit/eb4a3522373e0b4e2749b8a8bc965ff51355ea35 !

Could I ask that you split https://github.com/spesmilo/electrum/pull/8545/commits/3cabc987fece1387fa4b7e58c45ad4fcd1a10a4f into my changes and yours (e.g. by diffing the current commit with https://github.com/SomberNight/electrum/commit/79ff59e88b8989ba901e57e03f049e9d698eb2de)? So it's visible later in the git blame.

Reply:
Done